docs/articles/expensify-classic/domains/Create-a-group.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This file looks like it doesn't follow the same naming scheme. For instance we have `Create-A-Report.md` so I think this should be `Create-A-Group`

Reply:
Fixed